song/templates/backup-cronjob.yaml

```diff
@@ -1,59 +0,0 @@
-{{- if .Values.backup.enabled }}
-apiVersion: batch/v1beta1
-kind: CronJob
```

Comment:
What was the problem with this approach?

Reply:
it never really worked, and the docker image code is not checked in to github. it was pushed to dockerhub directly from someones laptop....

Reply:
so i created a simple spring boot app using springs internal Scheduling mechanism

Reply:
yeah it's cool to change the docker image, I assume you meant `song-postgres-backup-s3`, I'm just wondering about the CronJob because we are planning to use CronJob in platform backups